Travis coverage build failing

We're currently getting our coverage builds failing.  It looks like it's something about matplotlib.

```
__________ ERROR at setup of TestPSAnalysis.test_dendrogram_produced ___________
[gw1] linux -- Python 3.6.3 /home/travis/miniconda/envs/test/bin/python
self = <testsuite.MDAnalysisTests.analysis.test_psa.TestPSAnalysis object at 0x7fdae38e2550>
psa = <MDAnalysis.analysis.psa.PSAnalysis object at 0x7fdb546c5278>
    @pytest.fixture()
    def plot_data(self, psa):
        psa.run(metric='hausdorff')
        psa.run(metric='discrete_frechet')
>       return psa.plot()
testsuite/MDAnalysisTests/analysis/test_psa.py:79: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../../miniconda/envs/test/lib/python3.6/site-packages/MDAnalysis/analysis/psa.py:1700: in plot
    from matplotlib.pyplot import figure, colorbar, cm, savefig, clf
../../../miniconda/envs/test/lib/python3.6/site-packages/matplotlib/pyplot.py:29: in <module>
    import matplotlib.colorbar
../../../miniconda/envs/test/lib/python3.6/site-packages/matplotlib/colorbar.py:32: in <module>
    import matplotlib.artist as martist
../../../miniconda/envs/test/lib/python3.6/site-packages/matplotlib/artist.py:16: in <module>
    from .path import Path
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
    """
    
    from __future__ import (absolute_import, division, print_function,
                            unicode_literals)
    
    import six
    
    import math
    from weakref import WeakValueDictionary
    
    import numpy as np
    
>   from . import _path, rcParams
E   ImportError: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /home/travis/miniconda/envs/test/lib/python3.6/site-packages/matplotlib/_path.cpython-36m-x86_64-linux-gnu.so)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,4 +1,5 @@
 language: generic
+group: travis_latest
 
 # Only build for develop and master (and PRs)
 branches:
@@ -46,19 +47,19 @@ matrix:
   fast_finish: true
   include:
     - env: NAME="Doc"
-           MAIN_CMD="cd package && python setup.py"
-           SETUP_CMD="build_sphinx"
-           BUILD_DOCS=true
-           BUILD_CMD="cd ${TRAVIS_BUILD_DIR}/package && python setup.py build_ext --inplace"
-           INSTALL_HOLE="false"
+	   MAIN_CMD="cd package && python setup.py"
+	   SETUP_CMD="build_sphinx"
+	   BUILD_DOCS=true
+	   BUILD_CMD="cd ${TRAVIS_BUILD_DIR}/package && python setup.py build_ext --inplace"
+	   INSTALL_HOLE="false"
 
     - env: NAME="Lint"
-           PYLINTRC="${TRAVIS_BUILD_DIR}/package/.pylintrc"
-           MAIN_CMD="pylint package/MDAnalysis && pylint testsuite/MDAnalysisTests"
-           SETUP_CMD=""
-           BUILD_CMD=""
-           CONDA_DEPENDENCIES=""
-           INSTALL_HOLE="false"
+	   PYLINTRC="${TRAVIS_BUILD_DIR}/package/.pylintrc"
+	   MAIN_CMD="pylint package/MDAnalysis && pylint testsuite/MDAnalysisTests"
+	   SETUP_CMD=""
+	   BUILD_CMD=""
+	   CONDA_DEPENDENCIES=""
+	   INSTALL_HOLE="false"
 
     - os: osx
       env: PYTHON_VERSION=3.6
@@ -67,8 +68,8 @@ matrix:
       env: PYTHON_VERSION=2.7
 
     - env: NAME='minimal'
-           CONDA_DEPENDENCIES=${CONDA_MIN_DEPENDENCIES}
-           INSTALL_HOLE="false"
+	   CONDA_DEPENDENCIES=${CONDA_MIN_DEPENDENCIES}
+	   INSTALL_HOLE="false"
   allow_failures:
     - env: NUMPY_VERSION=dev EVENT_TYPE="cron"
 
@@ -77,9 +78,9 @@ install:
   # additional external tools (Issue #898) -- HOLE
   - |
     if [[ $INSTALL_HOLE == 'true' ]]; then \
-        bash ./maintainer/install_hole.sh $TRAVIS_OS_NAME "${HOME}"; \
-        HOLE_BINDIR="${HOME}/hole2/exe"; \
-        export PATH=${PATH}:${HOLE_BINDIR}; \
+	bash ./maintainer/install_hole.sh $TRAVIS_OS_NAME "${HOME}"; \
+	HOLE_BINDIR="${HOME}/hole2/exe"; \
+	export PATH=${PATH}:${HOLE_BINDIR}; \
     fi
   - git clone git://github.com/astropy/ci-helpers.git
   - source ci-helpers/travis/setup_conda.sh
@@ -98,5 +99,5 @@ after_success:
     fi
   # can't use test here since this leads to travis fails even though the build passes
   - if [[ ${TRAVIS_PULL_REQUEST} == "false" ]] && [[ ${BUILD_DOCS} == "true" ]] && [[ ${TRAVIS_BRANCH} == ${GH_DOC_BRANCH} ]]; then
-            bash ${TRAVIS_BUILD_DIR}/maintainer/deploy_docs.sh;
+	    bash ${TRAVIS_BUILD_DIR}/maintainer/deploy_docs.sh;
     fi

--- a/.travis.yml
+++ b/.travis.yml
@@ -47,19 +47,19 @@ matrix:
   fast_finish: true
   include:
     - env: NAME="Doc"
-	   MAIN_CMD="cd package && python setup.py"
-	   SETUP_CMD="build_sphinx"
-	   BUILD_DOCS=true
-	   BUILD_CMD="cd ${TRAVIS_BUILD_DIR}/package && python setup.py build_ext --inplace"
-	   INSTALL_HOLE="false"
+           MAIN_CMD="cd package && python setup.py"
+           SETUP_CMD="build_sphinx"
+           BUILD_DOCS=true
+           BUILD_CMD="cd ${TRAVIS_BUILD_DIR}/package && python setup.py build_ext --inplace"
+           INSTALL_HOLE="false"
 
     - env: NAME="Lint"
-	   PYLINTRC="${TRAVIS_BUILD_DIR}/package/.pylintrc"
-	   MAIN_CMD="pylint package/MDAnalysis && pylint testsuite/MDAnalysisTests"
-	   SETUP_CMD=""
-	   BUILD_CMD=""
-	   CONDA_DEPENDENCIES=""
-	   INSTALL_HOLE="false"
+           PYLINTRC="${TRAVIS_BUILD_DIR}/package/.pylintrc"
+           MAIN_CMD="pylint package/MDAnalysis && pylint testsuite/MDAnalysisTests"
+           SETUP_CMD=""
+           BUILD_CMD=""
+           CONDA_DEPENDENCIES=""
+           INSTALL_HOLE="false"
 
     - os: osx
       env: PYTHON_VERSION=3.6
@@ -68,8 +68,8 @@ matrix:
       env: PYTHON_VERSION=2.7
 
     - env: NAME='minimal'
-	   CONDA_DEPENDENCIES=${CONDA_MIN_DEPENDENCIES}
-	   INSTALL_HOLE="false"
+           CONDA_DEPENDENCIES=${CONDA_MIN_DEPENDENCIES}
+           INSTALL_HOLE="false"
   allow_failures:
     - env: NUMPY_VERSION=dev EVENT_TYPE="cron"
 
@@ -78,9 +78,9 @@ install:
   # additional external tools (Issue #898) -- HOLE
   - |
     if [[ $INSTALL_HOLE == 'true' ]]; then \
-	bash ./maintainer/install_hole.sh $TRAVIS_OS_NAME "${HOME}"; \
-	HOLE_BINDIR="${HOME}/hole2/exe"; \
-	export PATH=${PATH}:${HOLE_BINDIR}; \
+        bash ./maintainer/install_hole.sh $TRAVIS_OS_NAME "${HOME}"; \
+        HOLE_BINDIR="${HOME}/hole2/exe"; \
+        export PATH=${PATH}:${HOLE_BINDIR}; \
     fi
   - git clone git://github.com/astropy/ci-helpers.git
   - source ci-helpers/travis/setup_conda.sh
@@ -99,5 +99,5 @@ after_success:
     fi
   # can't use test here since this leads to travis fails even though the build passes
   - if [[ ${TRAVIS_PULL_REQUEST} == "false" ]] && [[ ${BUILD_DOCS} == "true" ]] && [[ ${TRAVIS_BRANCH} == ${GH_DOC_BRANCH} ]]; then
-	    bash ${TRAVIS_BUILD_DIR}/maintainer/deploy_docs.sh;
+            bash ${TRAVIS_BUILD_DIR}/maintainer/deploy_docs.sh;
     fi

--- a/.travis.yml
+++ b/.travis.yml
@@ -19,7 +19,7 @@ env:
     - GIT_CI_EMAIL=TravisCI@mdanalysis.org
     - MDA_DOCDIR=${TRAVIS_BUILD_DIR}/package/doc/html/html
     # Set default python version to avoid repetition later
-    - PYTHON_VERSION=3.6
+    - PYTHON_VERSION=3.5
     - BUILD_DOCS=false
     - COVERALLS=false
     - PYTEST_FLAGS="--disable-pytest-warnings --numprocesses 2 --durations=50"
@@ -37,7 +37,7 @@ env:
   matrix:
     # Run a coverage test
     - COVERALLS="true" SETUP_CMD="${PYTEST_FLAGS} --cov=MDAnalysis"
-    - PYTHON_VERSION=3.5
+    - PYTHON_VERSION=3.6
     - PYTHON_VERSION=3.4
     - PYTHON_VERSION=2.7
     - NUMPY_VERSION=1.10.4

